src/actions-on-google.ts

```diff
@@ -481,6 +483,8 @@ export class ActionsOnGoogle {
         config.getAudioOutConfig().setEncoding(1)
         config.getAudioOutConfig().setSampleRateHertz(16000)
         config.getAudioOutConfig().setVolumePercentage(100)
+        config.setScreenOutConfig(new embeddedAssistantPb.ScreenOutConfig())
+        config.getScreenOutConfig().setScreenMode('PLAYING')
```

Comment:
I don't want to necessarily require every request to have a `ScreenOutConfig` if it is not needed. Can this be turned into an option?

Reply:
Yes, will do.